zebrad/tests/acceptance.rs

```diff
@@ -2953,3 +2961,13 @@ fn scan_start_where_left() -> Result<()> {
 
     Ok(())
 }
+
+/// Test successful registration of a new key in the scan task.
+///
+/// See [`common::shielded_scan::register_key`] for more information.
+#[tokio::test]
+#[ignore]
+#[cfg(feature = "shielded-scan")]
+async fn scans_for_new_key() -> Result<()> {
```

Comment:
I am not sure if this test currently runs in the CI. iirc it needs some code to run. We can do this in another ticket.

Reply:
It doesn't yet, opened #8236 for improving this test and adding it to CI.